src/tools/spectest-interp.cc

```diff
@@ -703,7 +703,7 @@ wabt::Result JSONParser::ParseLaneConstValue(Type lane_type,
                                              ExpectedValue* out_value,
                                              string_view value_str,
                                              AllowExpected allow_expected) {
-  v128& v = out_value->value.value.v128_;
+  v128& v = out_value->value.value.GetRef<v128>();
```

Comment:
Maybe do `Get` here, then `Set` at the bottom instead, then we can remove `GetRef`.

Reply:
That's the first thing I had tried and for some reason it originally didn't work.. but now it does :)